sklearn/cluster/optics_.py

```diff
@@ -168,18 +168,23 @@ class OPTICS(BaseEstimator, ClusterMixin):
 
     Parameters
     ----------
-    min_samples : int
+    NOTE: should we call this parameter min_neighbors, or n_neighbors,
+    to be more consistent with the NearestNeighbors API, which this parameter
+    is used for?
+    min_samples : int (default=5)
```

Comment:
@jnothman sorry, I was talking about this parameter, which is different than `min_cluster_size`.

My comment here was:

> should we call this parameter min_neighbors, or n_neighbors,
>     to be more consistent with the NearestNeighbors API, which this parameter
>     is used for?

Reply:
Yep! Hadn't considered DBSCAN's terminology!